staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -399,6 +399,7 @@ type ListOptions struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // ExportOptions is the query options to the standard REST get call.
+// Deprecated
```

Comment:
add deprecated comments to the Export and Exact fields below, and those will get folded into the published openapi schema

Reply:
suggest specifying the version they will be removed in (1.18)